DEPS

```diff
@@ -8,7 +8,7 @@ deps = {
   "vendor/boto": "https://github.com/boto/boto@f7574aa6cc2c819430c1f05e9a1a1a666ef8169b",
   "vendor/python-patch": "https://github.com/brave/python-patch@d8880110be6554686bc08261766538c2926d4e82",
   "vendor/omaha":  "https://github.com/brave/omaha.git@cd2499ce23b2f8c532e1c504adf41d3e946780d2",
-  "vendor/sparkle": "https://github.com/brave/Sparkle.git@a8ba961ee53e7bf8484845ec14050b74f530e813",
+  "vendor/sparkle": "https://github.com/brave/Sparkle.git@72a4d6974083301c3e182940dbf36f8c744156bb",
```

Comment:
Commit 72a4d6 "does not belong to any branch" in the brave/Sparkle repository ([source](https://github.com/brave/Sparkle/commit/72a4d6974083301c3e182940dbf36f8c744156bb)). Is this intended?

Reply:
Well, not sure why it says like that. However, this is WIP PR and merge commit hash will be used when sparkle PR is merged.